docs/topics/hacking/debugging.rst

```diff
@@ -0,0 +1,47 @@
+=========
+Debugging
+=========
+
+The docker setup use supervisord to run the django runserver. This means
```

Comment:
You could link this back to the docker install docs like ``:ref:`docker setup <install-with-docker>```